pkg/splunk/enterprise/monitoringconsole.go

```diff
@@ -58,29 +58,6 @@ func ApplyMonitoringConsole(ctx context.Context, client splcommon.ControllerClie
 		return result, err
 	}
 
-	cr.Status.Selector = fmt.Sprintf("app.kubernetes.io/instance=splunk-%s-monitoring-console", cr.GetName())
```

Comment:
Why has this section of code been removed?

Reply:
This code is present twice and misplaced due to a merging issue. I see repetitive code here https://github.com/splunk/splunk-operator/blob/feature-appframework/pkg/splunk/enterprise/monitoringconsole.go#L71 and https://github.com/splunk/splunk-operator/blob/feature-appframework/pkg/splunk/enterprise/monitoringconsole.go#L112